src/pages/about-us/components/_hero.tsx

```diff
@@ -100,23 +99,23 @@ const Hero = () => {
                             alt="example"
                             width="100%"
                         />
-                        <FlexHeader>
-                            <StyledHeaderDesktopAbout
-                                as="h1"
-                                color="white"
-                                align="center"
-                                type="unset"
-                            >
-                                {localize('About us')}
-                            </StyledHeaderDesktopAbout>
-                        </FlexHeader>
+                        <StyledHeaderDesktopAbout
+                            as="h1"
+                            color="white"
+                            align="center"
+                            type="main-landing-title"
+                            width="100%"
+                            padding="0"
+                        >
+                            {localize('Who we are')}
+                        </StyledHeaderDesktopAbout>
```

Comment:
Since this is a custom title. Please just extend `h1` or `h2` directly instead of `header` and then overriding the styles.

Reply:
done